molecule/default/get_server_uptime.yml

```diff
@@ -0,0 +1,10 @@
+---
+# tasks file for molecule-validate-nova-deploy
+
+- name: Check server uptime
+  shell: uptime
```

Comment:
Since a shell environment is not necessary for this operation, the `command` module is preferable for this type of action.